src/olympia/addons/models.py

```diff
@@ -351,6 +351,45 @@ def get_needs_human_review_queue(self, admin_reviewer=False):
                 addonreviewerflags__needs_admin_code_review=True)
         return qs
 
+    def get_mad_queue(self):
+        qs = (
+            self.get_queryset()
+            # All valid statuses, plus incomplete as well because the add-on
+            # could be purely unlisted (so we can't use valid_q(), which
+            # filters out current_version=None). We know the add-ons are likely
+            # to have a version since they got the needs_human_review flag, so
+            # returning incomplete ones is acceptable.
+            .filter(
+                status__in=[
+                    amo.STATUS_APPROVED, amo.STATUS_NOMINATED, amo.STATUS_NULL
+                ],
+                versions__files__status__in=[
+                    amo.STATUS_APPROVED, amo.STATUS_AWAITING_REVIEW,
+                ],
+                versions__versionscannerflags__needs_human_review_by_mad=True
+            )
+            # We don't want the default transformer.
+            # See get_auto_approved_queue()
+            .only_translations()
+            # We need those joins for the queue to work without making extra
+            # queries. See get_auto_approved_queue()
+            .select_related(
+                'addonapprovalscounter',
+                'addonreviewerflags',
+                '_current_version__autoapprovalsummary',
+            )
+            .prefetch_related(
+                '_current_version__files'
+            )
+            .order_by(
+                'created',
+            )
+            # There could be several versions matching for a single add-on so
+            # we need a distinct.
+            .distinct()
+        )
```

Comment:
the other similar methods filter out addons that have `addonreviewerflags__needs_admin_code_review` (or content_review) - do we want to be including those to non-admin reviewers?  They can't do anything on the review page iirc, (can they even access the review page?).  Similarly with unlisted-only addons - there's a different permission needed for those reviews so can standard reviewers do anything?

If the review page can be accessed but they can't do anything that seems suboptimal, but if Product are okay with it, then sure.  Any inaccessible review page shouldn't be shown.

Reply:
Ha, so does that mean `addonreviewerflags__needs_admin_code_review` prevents regular reviewers to review add-on?

Reply:
there's been some changes on that recently iirc, so... @diox or @wagnerand ?

Reply:
`needs_admin_code_review`  prevents regular reviewers from reviewing code, yes. `needs_admin_content_review` prevents content reviewers from reviewing... content.

*technically*, if `needs_admin_content_review` is `True` regular reviewers can't review either, because they are supposed to do a full review, and that includes content. The queue list doesn't care about that, but the review page does.